routing/router.go

```diff
@@ -3081,13 +3081,12 @@ func (r *ChannelRouter) MarkEdgeLive(chanID lnwire.ShortChannelID) error {
 // channels that satisfy all requirements.
 type ErrNoChannel struct {
 	position int
-	fromNode route.Vertex
```

Comment:
It's mentioned in the commit message. So it sounds like we could extract that info at a later point, so we can log it? But in this commit it doesn't look like it yet. Still going through the PR, so will circle back to it.

Reply:
I removed it because the node pubkey isn't available later when encountering an error and it's redundant, one can figure out the problematic pub key by the index of the input keys. I could pass in the hops into the functions that generate those errors, to remain backward compatible if wanted?